LambdaSource/Scotty_TableSlotUpdater/lambdaHandler.py

```diff
@@ -1,10 +1,98 @@
 #! python3
 import boto3
-import argparse
-
+import time
 from pprint import pprint
 
 
+def updateBot(lex, updatedSlotVersion):
+    bot = lex.get_bot(
+        name='Scotty',
+        versionOrAlias="$LATEST"
+    )
+    intentVersion = []
+    intentName = []
+    for intent in bot['intents']:
+        newIntent = lex.get_intent(name=intent['intentName'], version="$LATEST")
+        for i in range(len(newIntent['slots'])):
```

Comment:
Instead of doing it this way, I'd probably do something like:
```Python
for slot in newIntent['slots']:
   if slot['name'] == "table":
      # do stuff
      break
```
Once you find it, you can break out of the loop and stop processing the rest of the items because there can only be one slot with that name.